inspirehep/utils/helpers.py

```diff
@@ -78,3 +78,12 @@ def force_force_list(data):
     elif isinstance(data, (tuple, set)):
         return list(data)
     return data
+
+def is_deleted(data):
```

Comment:
This is not worth to be put in `.helpers`. You can embed it directly into the `collection` rule.